docs/reference/migration/migrate_5_0/allocation.asciidoc

```diff
@@ -39,6 +39,14 @@ command corresponds to the old `allocate` command  with `allow_primary` set to
 false. The new `allocate_empty_primary` command corresponds to the old
 `allocate` command with `allow_primary` set to true.
 
+==== Custom Reroute Commands
+
+Elasticsearch no longer supports plugins registering custom allocation
+commands. It didn't look like it fully worked anyway and when we went to get
+it working and tested we realized it was a fairly scary thing for a plugin to
+do. If this is really really a thing your plugin needs to do then open a
+github issue.
```

Comment:
can we have a bit less information here? Just say it was unused and probably not needed.